sbc_translation/security/ir.model.access.csv

```diff
@@ -12,4 +12,5 @@ access_admin_translator_competence,Full access on translator competence,model_tr
 access_translation_competence,Read access on translation competence,model_translation_competence,group_user,1,0,0,0
 access_admin_translation_competence_user,Full access on translation competence,model_translation_competence,group_manager,1,1,1,1
 access_model_sbc_reply_to_comments_wizard,Full access to reply to comments,model_sbc_reply_to_comments_wizard,group_manager,1,1,1,1
-access_model_sbc_reply_to_issue_wizard,Full access to reply to issue,model_sbc_reply_to_issue_wizard,group_manager,1,1,1,1
\ No newline at end of file
+access_model_sbc_reply_to_issue_wizard,Full access to reply to issue,model_sbc_reply_to_issue_wizard,group_manager,1,1,1,1
+access_model_translation_letter_counting_wizard,Access on Translation letter counting,model_translation_letter_counting_wizard,group_user,1,1,1,1
```

Comment:
As discussed, make sure the Base User has access to it, not the Translation Platform / User